projects/common/src/color/color-palette.ts

```diff
@@ -16,6 +16,10 @@ export class ColorPalette {
       return [...this.basisColors];
     }
 
+    if (count === 1) {
+      return [this.basisColors[0]];
+    }
+
     if (count === 2) {
       // Use the middle and end colors for generating 2 colors from a larger palette
       return this.forNColors(3).slice(0, 2);
```

Comment:
If we're reversing the palettes such that we're selecting from the LHS first (which I think makes sense), the 2 count case should swap too, to use start and middle colors.

Reply:
I think the comment is wrong

It's doing `.slice(0, 2)`, which means it's using the first 2 colors from a larger palette

Reply:
It does look that way doesn't it? and the test seems to confirm.
```
  test('should return first and middle color if two colors requested from a larger palette', () => {
    const colors = ['rgb(0, 0, 0)', 'rgb(50, 50, 50)', 'rgb(100, 100, 100)'];
    expect(new ColorPalette(colors).forNColors(2)).toEqual([colors[0], colors[1]]);
  });
```

Could you update that comment then?

Reply:
yep.